### PR TITLE
8282201: Consider removal of expiry check in VerifyCACerts.java test

### PR DESCRIPTION
**Pull request by rhalade**
This fix removes the check for 90 days expiry. But the test will continue to fail if the expired certificate is included in `cacerts` but no exception is granted.

Fix also includes minor code cleanup.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8282201](https://bugs.openjdk.org/browse/JDK-8282201): Consider removal of expiry check in VerifyCACerts.java test

### Reviewers
 * [Xue-Lei Andrew Fan](https://openjdk.org/census#xuelei) (@XueleiFan - **Reviewer**) ⚠️ Review applies to [070b4124](https://git.openjdk.org/jdk/pull/12910/files/070b41240a662232ec2993243539a7cac228b01a)
 * [Sean Mullan](https://openjdk.org/census#mullan) (@seanjmullan - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12910/head:pull/12910` \
`$ git checkout pull/12910`

Update a local copy of the PR: \
`$ git checkout pull/12910` \
`$ git pull https://git.openjdk.org/jdk pull/12910/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12910`

View PR using the GUI difftool: \
`$ git pr show -t 12910`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12910.diff">https://git.openjdk.org/jdk/pull/12910.diff</a>

</details>

### CODE REVIEW

File: test/jdk/sun/security/lib/cacerts/VerifyCACerts.java [hunk 1]
```diff
@@ -313,66 +315,56 @@ public static void main(String[] args) throws Exception {
...
-                        + e.getMessage());
+                System.err.println("ERROR: cert cannot be verified:" + e.getMessage());
             }
 
-            // Make sure cert is not expired or not yet valid
+            // Is cert expired?
             try {
                 cert.checkValidity();
             } catch (CertificateExpiredException cee) {
                 if (!EXPIRY_EXC_ENTRIES.contains(alias)) {
```
Comment (rhalade) [inline test/jdk/sun/security/lib/cacerts/VerifyCACerts.java:345]: @seanjmullan 90 days expiry check is removed but test will continue to check expiry exception. This allows extra time to consider removal of root.

**Comment by XueleiFan** [comment]
If the check get removed, how could the expired cert get caught in the future?

**Comment by rhalade** [comment]
> 

The fix removes the early warning (90 days). Usually this is very early and only fix is to add certificate to expiry exception. Test would still fail when a expired certificate is included but no exception is granted. I have filed tracking bugs for next few years with due date so we don't miss any.

**Review by XueleiFan — APPROVED** [review]

**Review by seanjmullan — APPROVED** [review]

**Pull request closed without merging**